requirements/bench.txt

```diff
@@ -13,3 +13,4 @@ tqdm
 torchvision
 pyomo>=5.7
 IOHexperimenter>=0.2.8
+mixsimulator
```

Comment:
can you also add a version number please, that's helpful to identify which versions has been initially used.